packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -0,0 +1,237 @@
+import {Component, h, State, Prop, VNode, Host} from '@stencil/core';
+import {
+  NumericFacet,
+  buildNumericFacet,
+  NumericFacetState,
+  NumericFacetOptions,
+  RangeFacetSortCriterion,
+  SearchStatus,
+  SearchStatusState,
+  buildSearchStatus,
+  NumericFacetValue,
+  NumericFilterState,
+  NumericFilter,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {FacetPlaceholder} from '../../facets/atomic-facet-placeholder/atomic-facet-placeholder';
+import {FacetContainer} from '../facet-container/facet-container';
+import {FacetHeader} from '../facet-header/facet-header';
+import {FacetValueCheckbox} from '../facet-value-checkbox/facet-value-checkbox';
+import {FacetValueLink} from '../facet-value-link/facet-value-link';
+import {BaseFacet} from '../facet-common';
+
+/**
+ * A facet is a list of values for a certain field occurring in the results, ordered using a configurable criteria (e.g., number of occurrences).
+ * An `atomic-rating-facet` displays a facet of the results for the current query as ratings.
+ *
+ * @part facet - The wrapper for the entire facet.
+ * @part placeholder - The placeholder shown before the first search is executed.
+ *
+ * @part label-button - The button that displays the label and allows to expand/collapse the facet.
+ * @part label-button-icon - The label button icon.
+ * @part clear-button - The button that resets the actively selected facet values.
+ * @part clear-button-icon - The clear button icon.
+ *
+ * @part values - The facet values container.
+ * @part value-label - The facet value label, common for all displays.
+ * @part value-count - The facet value count, common for all displays.
+ *
+ * @part value-checkbox - The facet value checkbox, available when display is 'checkbox'.
+ * @part value-checkbox-label - The facet value checkbox clickable label, available when display is 'checkbox'.
+ * @part value-link - The facet value when display is 'link'.
+ * @part value-box - The facet value when display is 'box'.
+ *
+ * @part show-more - The show more results button.
+ * @part show-less - The show less results button.
+ * @part show-more-less-icon - The icons of the show more & show less buttons.
+ */
+@Component({
+  tag: 'atomic-rating-facet',
+  styleUrl: 'atomic-rating-facet.pcss',
+  shadow: true,
+})
+export class AtomicRatingFacet
+  implements
+    InitializableComponent,
+    BaseFacet<NumericFacet, NumericFacetState> {
+  @InitializeBindings() public bindings!: Bindings;
+  public facet!: NumericFacet;
+  public filter?: NumericFilter;
```

Comment:
filter/filterState aren't needed, they are used for the custom input in the other Numeric Facet